packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -1194,7 +1211,7 @@ class VirtualListBasic extends Component {
 		return (
 			<div className={containerClasses} data-webos-voice-focused={voiceFocused} data-webos-voice-group-label={voiceGroupLabel} data-webos-voice-disabled={voiceDisabled} ref={this.props.scrollContentRef} style={style}>
 				<div {...rest} className={contentClasses} ref={this.contentRef}>
-					{itemsRenderer({cc, itemContainerRef, primary})}
+					{itemsRenderer({cc, primary})}
```

Comment:
I think we are referring to `itemContainerRef` in sandstone VL.
Please make a change in sandstone along with this.

Reply:
Fix it in the sandstone PR and in the agate PR.